src/components/bottom-sheet/AnimatedBottomSheet.tsx

```diff
@@ -1,14 +1,21 @@
-import {Animated, LayoutChangeEvent, useWindowDimensions} from 'react-native';
+import {
+  Animated,
+  LayoutChangeEvent,
+  useWindowDimensions,
+  ViewStyle,
+} from 'react-native';
 import React, {ReactNode, useMemo} from 'react';
 import {StyleSheet} from '@atb/theme';
 
 export default function AnimatedBottomSheet({
   animatedOffset,
   children,
   onLayout,
+  style,
 }: {
   animatedOffset: Animated.Value;
   children: ReactNode;
+  style?: ViewStyle;
```

Comment:
If we can avoid it, I would like to not be able to pass `style` from outside. This makes it easier to maintain the same appearance for all bottom-sheets in the app.

Reply:
Fixed